runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -107,4 +108,15 @@ default int getOxygenThreshold()
 	{
 		return 0;
 	}
+
+	@ConfigItem(
+			keyName = "spec",
+			name = "Special Attack Energy Notification Threshold",
+			position = 7,
+			description = "The amount of spec energy naturally regenerated to send a notification at. A value of 0 will disable notification."
```

Comment:
You should shorten this up or add line breaks.

Reply:
Would dropping the word 'naturally' be sufficient?